app/builders/immediate_email_builder.rb

```diff
@@ -1,43 +1,57 @@
 class ImmediateEmailBuilder
-  def initialize(subscriber:, content_change:)
-    @subscriber = subscriber
-    @content_change = content_change
+  def initialize(subscriber_content_changes)
+    @subscriber_content_changes = subscriber_content_changes
```

Comment:
Probably overkill but as I understand this is a an array of hashes right? I wonder if it'd be consistent with the digest equivalent to have it be an array of structs (that might not even be how that works - ignore me if I've got the scenarios wrong)

Reply:
The digest equivalent only makes one email, so it doesn't need an array, it uses keyword arguments instead. So in a way this implement is more consistent with digests.

We might want to revisit this if we decide that the digest equivalent needs to also create things in bulk.